config/peerpods/podvm/libvirt-podvm-image-cm.yaml

```diff
@@ -25,4 +25,6 @@ data:
 
   # To Enable SE for IBM Z
   SE_BOOT: "true"
-  
+
+  # For Pre-built PodVM images.
+  PODVM_IMAGE_URI: "oci::quay.io/openshift_sandboxed_containers/libvirt-podvm-image:latest::/image/podvm.qcow2"
```

Comment:
I just wanted to provide an example, probably will add it as a comment, so that it won't be the default. but again from a libvirt PoV, there are a bunch of params including `ORG_ID`, `ACTIVATION_KEY` on `libvirt-podvm-image-cm` which are required for both `operator-built` and `pre-built` scenarios. so it's mandatory that user should create/apply this cm and probably not the operator creating it.